wordpress-mobile/WordPress-iOS
# Preliminary implementation of Selected Post Summary Chart View

Fixes #11062, which calls for the creation of a chart for a selected post. We don't yet have a _Post Details_ screen, so I'm using the _Latest Post Summary_ to scaffold and demonstrate these changes.

To test:

- Checkout the branch, confirm that it builds, and that existing tests pass.
- Manually run the app and visually confirm that a site with a recent post still looks just like the video in #11109.
- Now manually revert commit `7d28d9f`, run the app, and confirm that it looks a bit more like _Exhibit A_ below.

Update release notes:


<details>
<summary>Exhibit A</summary>

![11062a_pr](https://user-images.githubusercontent.com/221062/53605416-5203d380-3b6c-11e9-927d-1b4cbe1fce7c.gif)

</details>
